hosts.inventory

```diff
@@ -0,0 +1,13 @@
+# The `hosts` inventory file lists the JupyterHub servers managed by Ansible
+
+# This provides an inventory of host servers used for JupyterHub
+# Edit the fqdn (fully qualified domain name) for your hub server
+# For example:
+#
+# [jupyterhub_hosts]
+# www.example.com
+#
+# Save file as `hosts` (without the file type suffix) when done editing.
+
+[jupyterhub_hosts]
+fqdn.goes.here
```

Comment:
Can this go back to the `.example` name? It's a standard pattern for examples of files that need copying. I think the  `.inventory` here suggests that the file would be used if it were edited without being copied to the `hosts` name, whereas `hosts.example` indicates that it is only an example file, and its contents are ignored. The content is a great addition, explaining this for those who may not be familiar with example files.